apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCardButtons.jsx

```diff
@@ -161,6 +175,16 @@ class ActivitySectionCardButtons extends Component {
               <i style={{marginRight: 7}} className="fa fa-plus-circle" />
               Resource Link
             </button>
+            <button
+              onMouseDown={this.handleOpenAddVocabulary}
+              className="btn"
+              style={styles.addButton}
+              type="button"
+            >
+              <i style={{marginRight: 7}} className="fa fa-plus-circle" />
+              Vocabulary Link
```

Comment:
I think to save space we should have this button just say "Vocab" and probably want to change Resource Link to Resource. There are just too many buttons...

Reply:
Makes sense. I guess the only reason I could see to keep "Vocabulary Link" is that this button adds a link, not a vocab, but I think that'll be pretty clear.